devops/yarn/src/main/java/org/ray/on/yarn/Log4jPropertyHelper.java

```diff
@@ -0,0 +1,37 @@
+package org.ray.on.yarn;
```

Comment:
nit: this package name looks a little weird. I think, `org.ray.yarn` may look better.

Reply:
Ok, It figures. I will discuss this package name in our team.